docs/src/index.md

```diff
@@ -33,6 +34,176 @@ plot!(ftm7, label="7th order")
+
+```@example range_bound_1d
+s = texp + tsin
+```
+
+```@example range_bound_1d
+p = texp * tsin
+```
+
+To bound the range of a Taylor model in one variable, use the function `bound_taylor1`:
+
+```@example range_bound_1d
+using TaylorModels: bound_taylor1
+
+[bound_taylor1(x) for x in [s, p]]
+```
+
+This shows in particular that $0.12499 \leq e^t + \sin(t) \leq 2.12501$ and that
+$-0.291667 \leq e^t \sin(t) \leq 0.791667$ for all $t \in D$. Such bounds are in general not tight.
+If desired, the common approach to improve the bounds is to evaluate the Taylor model
+on a smaller interval, e.g.
+
+```@example range_bound_1d
+D = domain(s) # domain -0.5 .. 0.5
+
+E = evaluate(s, D) # original, no mincing
+```
+
+```@example range_bound_1d
+Dm = mince(D, 8) # split the domain into 8 smaller chunks
+
+Em = evaluate.(s, Dm) # evaluate the Taylor model on each sub-domain
+
+Rm = reduce(hull, Em) # take the convex hull, i.e. the smallest interval that contains them all
+```
+
+Here the lower bound has been improved by mincing (or splitting) the domain, and it
+may improve by repeating such operation recursively on smaller domains.
+In particular, the fact that the lower bound is greater than zero constitutes an
+algorithmic proof that $s : t \mapsto e^t + \sin(t)$ is positive on $D$.
+Let's visualize the function $s(t)$ and the bounds obtained so far.
+
+```@example range_bound_1d
+using Plots
+
+Dt = range(-0.5, 0.5, length=100)
+
+fig = plot(xlab="t", ylab="s(t)", legend=:topleft)
+plot!(fig, Dt, t -> exp(t) + sin(t), lab="", c=:black)
+
+# range bounds
+plot!(fig, Dt, t -> sup(E), lab="N = 1", c=:blue, style=:dash)
+plot!(fig, Dt, t -> inf(E), c=:blue, lab="", style=:dash)
+
+plot!(fig, Dt, t -> sup(Rm), lab="N = 8", c=:red, style=:dash)
+plot!(fig, Dt, t -> inf(Rm), c=:red, lab="", style=:dash)
+
+R16 = reduce(hull, evaluate.(s, mince(D, 16)))
+plot!(fig, Dt, t -> sup(R16), lab="N = 16", c=:orange, style=:dash)
+plot!(fig, Dt, t -> inf(R16), c=:orange, lab="", style=:dash)
+```
+
+## Internal representation
+
+Consider again the Taylor model variable from the [Taylor model variables and range bounding](@ref) example.
+
+```@example range_bound_1d
+t
+```
+
+Such constructor is an alias for
+
+```julia
+    TaylorModel1(x0 + Taylor1(eltype(x0), ord), zero(dom), interval(x0), dom)
+```
+
+Taylor models in one variable are internally represented using four fields: a Taylor
+series (`pol`) in one variable that holds the polynomial approximation of order `ord`;
+the interval remainder (`rem`); the expansion point (`x0`), and the interval domain of
+interest (`dom`). Getter functions are defined for each of these fields:
+
+```@example range_bound_1d
+get_order(t)
+```
+
+```@example range_bound_1d
+remainder(t)
+```
+
+```@example range_bound_1d
+polynomial(t)
+```
+
+```@example range_bound_1d
+domain(t)
+```
+
+```@example range_bound_1d
+expansion_point(t)
+```
+
+Finally, note that the Taylor model type is a parametric structure with two fields,
```

Comment:
here `field` is wrong, the correct way to say is probably "is a structure with two parameters ..."

Reply:
What about "types"?